ui/src/dashboards/reducers/cellEditorOverlay.js

```diff
@@ -36,12 +42,15 @@ export default function cellEditorOverlay(state = initialState, action) {
         initializeOptions('table')
       )
 
+      const lineColors = validateLineColors(colors)
```

Comment:
What does this return when one of the other graph types is selected?

Reply:
I'm noticing that `initializeOptions` is probably a better approach. Also has more accurate name. I'll make a note to come back and consolidate this stuff later